### ISSUE TITLE
Wrong position of SIMPLE_IDENTIFIER when using 'global'

### ISSUE THREAD

**Issue by EgorBu** [bblfsh/python-driver#77]
Hi,
the position of SIMPLE_IDENTIFIER has an error when using `global` context.
ex:
```
abc = None


def b():
    global abc
    pass

```

Vis:
![selection_008](https://user-images.githubusercontent.com/2606702/29325507-3b203cc4-81e8-11e7-8a62-9ce708c072df.png)

**Comment by abeaumont** [comment]
Thanks for the report @EgorBu. Please note that you can use GH's markdown syntax highlighting instead of attaching an image of the code. Could you attach the generated UAST instead and indicate which part you think is wrong?

**Comment by EgorBu** [comment]
UAST:
```
uast {
  internal_type: "Module"
  children {
    internal_type: "Assign"
    properties {
      key: "internalRole"
      value: "body"
    }
    children {
      internal_type: "Name"
      properties {
        key: "ctx"
        value: "Store"
      }
      properties {
        key: "internalRole"
        value: "targets"
      }
      token: "abc"
      start_position {
        line: 1
        col: 1
      }
      end_position {
        offset: 2
        line: 1
        col: 3
      }
      roles: ASSIGNMENT_VARIABLE
      roles: SIMPLE_IDENTIFIER
      roles: EXPRESSION
    }
    children {
      internal_type: "NoneLiteral"
      properties {
        key: "internalRole"
        value: "value"
      }
      token: "<nil>"
      start_position {
        offset: 6
        line: 1
        col: 7
      }
      end_position {
        offset: 9
        line: 1
        col: 10
      }
      roles: NULL_LITERAL
      roles: EXPRESSION
      roles: ASSIGNMENT_VALUE
    }
    start_position {
      line: 1
      col: 1
    }
    end_position {
      offset: 9
      line: 1
      col: 10
    }
    roles: ASSIGNMENT
    roles: EXPRESSION
  }
  children {
    internal_type: "FunctionDef"
    properties {
      key: "internalRole"
      value: "body"
    }
    properties {
      key: "returns"
      value: "<nil>"
    }
    children {
      internal_type: "arguments"
      properties {
        key: "internalRole"
        value: "args"
      }
      properties {
        key: "kwarg"
        value: "<nil>"
      }
      properties {
        key: "vararg"
        value: "<nil>"
      }
      start_position {
        offset: 13
        line: 4
        col: 1
      }
      roles: FUNCTION_DECLARATION_ARGUMENT
      roles: INCOMPLETE
    }
    children {
      internal_type: "FunctionDef.body"
      properties {
        key: "promotedPropertyList"
        value: "true"
      }
      children {
        internal_type: "Global"
        children {
          internal_type: "Name"
          properties {
            key: "internalRole"
            value: "names"
          }
          token: "abc"
          start_position {
            offset: 25
            line: 5
            col: 4
          }
          roles: SIMPLE_IDENTIFIER
          roles: EXPRESSION
        }
        children {
          internal_type: "PreviousNoops"
          properties {
            key: "internalRole"
            value: "noops_previous"
          }
          children {
            internal_type: "NoopLine"
            properties {
              key: "internalRole"
              value: "lines"
            }
            token: "\n"
            start_position {
              offset: 11
              line: 2
              col: 1
            }
            roles: COMMENT
          }
          children {
            internal_type: "NoopLine"
            properties {
              key: "internalRole"
              value: "lines"
            }
            token: "\n"
            start_position {
              offset: 12
              line: 3
              col: 1
            }
            roles: COMMENT
          }
          start_position {
            offset: 11
            line: 2
            col: 1
          }
          end_position {
            offset: 12
            line: 3
            col: 1
          }
          roles: WHITESPACE
        }
        start_position {
          offset: 26
          line: 5
          col: 5
        }
        end_position {
          offset: 12
          line: 3
          col: 1
        }
        roles: STATEMENT
        roles: VISIBLE_FROM_WORLD
        roles: INCOMPLETE
      }
      children {
        internal_type: "Pass"
        token: "pass"
        start_position {
          offset: 41
          line: 6
          col: 5
        }
        end_position {
          offset: 44
          line: 6
          col: 8
        }
        roles: NOOP
        roles: STATEMENT
      }
      roles: FUNCTION_DECLARATION_BODY
    }
    token: "b"
    start_position {
      offset: 17
      line: 4
      col: 5
    }
    end_position {
      offset: 44
      line: 6
      col: 8
    }
    roles: FUNCTION_DECLARATION
    roles: FUNCTION_DECLARATION_NAME
    roles: SIMPLE_IDENTIFIER
  }
  start_position {
    line: 1
    col: 1
  }
  end_position {
    offset: 44
    line: 6
    col: 8
  }
  roles: FILE
}
```
Node with problem:
```
token: "abc"
          start_position {
            offset: 25
            line: 5
            col: 4
          }
          roles: SIMPLE_IDENTIFIER
          roles: EXPRESSION
```
it has wrong position (the screenshot shows what exactly wrong)

**Comment by abeaumont** [comment]
Depends on https://github.com/bblfsh/sdk/issues/163

**Comment by juanjux** [comment]
Fixed by #87.

**Comment by juanjux** [comment]
The fix will be in v0.9.0.
